Config model needs some dusting, too:
```
14:39:37 [ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.12.1:compile (default-compile) on project config-model: Compilation failure
14:39:37 [ERROR] /sd/workspace/src/github.com/vespa-engine/vespa/config-model/src/main/java/com/yahoo/vespa/model/content/storagecluster/FileStorProducer.java:[77,16] cannot find symbol
14:39:37 [ERROR]   symbol:   method use_per_document_throttled_delete_bucket(boolean)
14:39:37 [ERROR]   location: variable builder of type com.yahoo.vespa.config.content.StorFilestorConfig.Builder
```

--- a/config-model/src/main/java/com/yahoo/vespa/model/admin/metricsproxy/MetricsProxyContainerCluster.java
+++ b/config-model/src/main/java/com/yahoo/vespa/model/admin/metricsproxy/MetricsProxyContainerCluster.java
@@ -203,7 +203,7 @@ public class MetricsProxyContainerCluster extends ContainerCluster<MetricsProxyC
 
     private Optional<String> getSystemName() {
         Monitoring monitoring = getMonitoringService();
-        return monitoring != null && ! monitoring.getClustername().equals("") ?
+        return monitoring != null && !monitoring.getClustername().isEmpty() ?
                 Optional.of(monitoring.getClustername()) : Optional.empty();
     }
 

--- a/config-model/src/main/java/com/yahoo/vespa/model/content/storagecluster/FileStorProducer.java
+++ b/config-model/src/main/java/com/yahoo/vespa/model/content/storagecluster/FileStorProducer.java
@@ -74,7 +74,6 @@ public class FileStorProducer implements StorFilestorConfig.Producer {
         builder.num_response_threads(responseNumThreads);
         builder.response_sequencer_type(responseSequencerType);
         builder.use_async_message_handling_on_schedule(useAsyncMessageHandlingOnSchedule);
-        builder.use_per_document_throttled_delete_bucket(true);
         var throttleBuilder = new StorFilestorConfig.Async_operation_throttler.Builder();
         builder.async_operation_throttler(throttleBuilder);
     }

